src/component/admin/groups/GroupUserRoleCell/GroupUserRoleCell.tsx

```diff
@@ -35,8 +21,12 @@ export const GroupUserRoleCell = ({
     const renderBadge = () => (
         <ConditionallyRender
             condition={value === Role.Member}
-            show={<StyledBadge>{capitalize(value)}</StyledBadge>}
-            elseShow={<StyledOwnerBadge>{capitalize(value)}</StyledOwnerBadge>}
+            show={<Badge>{capitalize(value)}</Badge>}
+            elseShow={
+                <Badge color="success" icon={<StyledPopupStar />}>
```

Comment:
We might not want `icon={<StyledPopupStar />}` in this case. 
Can we double check with @NicolaeUnleash just to be sure?

Reply:
Yep we changed it everywhere